en/docs/deploy-and-publish/publish-on-dev-portal/publish-to-multiple-external-api-stores.md

```diff
@@ -47,7 +48,7 @@ In this guide, let's use two separate instances of WSO2 API Manager, and let's p
 
 5.  Navigate to  **Main > Resources > Browse**.
 
-    ![Menu option to navigate to the Browse option]({{base_path}}/assets/img/learn/navigate-main-resources.png)
+    <img src="{{base_path}}/assets/img/learn/navigate-main-resources.png" alt="Menu option to navigate to the Browse option" title="Menu option to navigate to the Browse option" width="60%" />
```

Comment:
Add the zoom option for all images that you edit pls. This is important when viewing docs on lower resolutions.